tests/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.3'
+services:
+  jahia:
+    image: "${JAHIA_IMAGE}"
+    container_name: jahia
+    ports:
+      - "${JAHIA_PORT}:8080"
+    extra_hosts:
+      - jahia:127.0.0.1
+    networks:
+      - stack      
+  # Maven container
+  maven:
+    image: "${TESTS_IMAGE}"
+    container_name: e2e-tests
+    depends_on:
+      - jahia
+    environment:
+      - MANIFEST=${MANIFEST}
+      - JAHIA_CONTEXT=${JAHIA_CONTEXT}
+      - JAHIA_USERNAME=${JAHIA_USERNAME}
+      - JAHIA_PASSWORD=${JAHIA_PASSWORD}
+      - JAHIA_HOST=${JAHIA_HOST}
+      - JAHIA_PORT=${JAHIA_PORT}
+      - NEXUS_USERNAME=${NEXUS_USERNAME}
+      - NEXUS_PASSWORD=${NEXUS_PASSWORD}
+      - NEXUS_INTERNAL_URL=${NEXUS_INTERNAL_URL}
+      - TEST_URL=${TEST_URL}
```

Comment:
At the compose level, could we use `JAHIA_URL` instead of `TEST_URL` ? This should be the base URL to connect to jahia (incl. with context when applicable)

Reply:
If I use JAHIA_URL, I can also remove JAHIA_PORT, JAHIA_HOST and JAHIA_CONTEXT

Reply:
Yes for sure for `JAHIA_CONTEXT`
For JAHIA_PORT and JAHIA_HOST it depends if you need to access jahia through those at some point (in the tests for example), if not then no need to keep it.